feature/tpm/tpm.go

```diff
@@ -205,6 +204,19 @@ func (s *tpmStore) writeSealed() error {
 	return atomicfile.WriteFile(s.path, buf, 0600)
 }
 
+func (s *tpmStore) All() iter.Seq2[ipn.StateKey, []byte] {
+	return func(yield func(ipn.StateKey, []byte) bool) {
+		s.mu.Lock()
```

Comment:
One subtlety here is that if the body of the range using this iterator attempts anything requiring the lock, the goroutine will deadlock. That doesn't seem to be an obvious problem in the existing usage, but maybe we ought to doc comment this method.

Reply:
Oh, nevermind, I see you already addressed this in another comment.